src/node/internal/internal_fs_promises.ts

```diff
@@ -226,7 +231,8 @@ export class FileHandle extends EventEmitter {
     return await readFile(this.#fd, options);
   }
 
-  readLines(_options: any = undefined): void {
+  readLines(_options: CreateReadStreamOptions = {}): void {
+    // TODO(node-fs): Implement
```

Comment:
Can you be a little bit more specific

Reply:
Actually, this one is going to be more in depth, the node.js impl depends on the readline module that we don't implement yet.